web/src/lib/components/asset-viewer/detail-panel.svelte

```diff
@@ -410,7 +425,31 @@
         <div><Icon path={mdiCameraIris} size="24" /></div>
 
         <div>
-          <p>{asset.exifInfo.make || ''} {asset.exifInfo.model || ''}</p>
+          <p>
+            <a
+              href={getCameraSearchHref(asset)}
+              title={$t('search_for_camera')}
+              class="hover:dark:text-immich-dark-primary hover:text-immich-primary"
+            >
+              {asset.exifInfo.make || ''}
+              {asset.exifInfo.model || ''}
+            </a>
```

Comment:
If neither the make nor model are set, will this still render the `a`? Seems like you'd want to do the check higher up, similar to line 439.